certified-connectors/AgilePoint NX/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1630 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "AgilePoint NX",
+    "description": "Use AgilePoint NX API to work with process model and human tasks and perform various business process management actions.",

```

Comment:
@agilepointapps This description seems from API prospective, Can you please update it from connector prospective which should describe concisely the main purpose and value offered by your connector.

Refer- [Documentation](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#description)

Reply:
@sriyen-msft I have done the changes as you suggested